content/glossary/sql/alter-table.md

```diff
@@ -0,0 +1,37 @@
+<!--
+---
+Title: "ALTER TABLE"
+Subjects:
+  - "sql"
```

Comment:
We just today removed the language names from the tags and subjects lists, since the language is already reflected in the file structure. So technically the two "sql"s should be removed, but there's no way you could have known that--that's on me

Reply:
👍🏼